rust_snuba/src/processors/outcomes.rs

```diff
@@ -0,0 +1,125 @@
+use std::collections::HashSet;
+
+use crate::types::{BadMessage, BytesInsertBatch, KafkaMessageMetadata};
+use rust_arroyo::backends::kafka::types::KafkaPayload;
+use serde::Deserialize;
+
+pub fn process_message(
+    payload: KafkaPayload,
+    _metadata: KafkaMessageMetadata,
+) -> Result<BytesInsertBatch, BadMessage> {
+    if let Some(payload_bytes) = payload.payload {
```

Comment:
```suggestion
    let payload_bytes = payload.payload.ok_or(BadMessage)?;
```

I recently landed a similar change to the existing processors

Reply:
Updated, thanks. Quick question: do we not need to log here?

Reply:
I don't think you need to log inside any message processors, just return the error and the caller will log it for you